pyinstaller.spec

```diff
@@ -4,52 +4,64 @@ import sys, site, os
 from path import Path
 
 block_cipher = None
+spyder_data = Path(site.getsitepackages()[-1]) / "spyder"
+parso_grammar = Path(site.getsitepackages()[-1]) / "parso/python/grammar36.txt"
 
-spyder_data = Path(site.getsitepackages()[-1]) / 'spyder'
-parso_grammar = Path(site.getsitepackages()[-1]) / 'parso/python/grammar36.txt'
-
-if sys.platform == 'linux':
-    oce_dir = Path(sys.prefix) / 'share' / 'oce-0.18'
+if sys.platform == "linux" or sys.platform == "darwin":
+    oce_dir = Path(sys.prefix) / "share" / "oce-0.18"
 else:
-    oce_dir = Path(sys.prefix) / 'Library' / 'share' / 'oce'
-
-a = Analysis(['run.py'],
-             pathex=['/home/adam/cq/CQ-editor'],
-             binaries=[],
-             datas=[(spyder_data ,'spyder'),
-                    (parso_grammar, 'parso/python'),
-                    (oce_dir , 'oce')],
-             hiddenimports=['ipykernel.datapub'],
-             hookspath=[],
-             runtime_hooks=['pyinstaller/pyi_rth_occ.py',
-                            'pyinstaller/pyi_rth_fontconfig.py'],
-             excludes=['_tkinter',],
-             win_no_prefer_redirects=False,
-             win_private_assemblies=False,
-             cipher=block_cipher,
-             noarchive=False)
-
-pyz = PYZ(a.pure, a.zipped_data,
-             cipher=block_cipher)
-exe = EXE(pyz,
-          a.scripts,
-          [],
-          exclude_binaries=True,
-          name='CQ-editor',
-          debug=False,
-          bootloader_ignore_signals=False,
-          strip=False,
-          upx=True,
-          console=True,
-          icon='icons/cadquery_logo_dark.ico')
-          
-exclude = ('libGL','libEGL','libbsd')
+    oce_dir = Path(sys.prefix) / "Library" / "share" / "oce"
+
+
+a = Analysis(
```

Comment:
What's with the formatting change? I think only one line is actually edited.

Reply:
The entire pyinstaller.spec file was accidentally reformatted with black.  Therefore, spacing and single-quote substitution was applied globally.  Since it had no semantic impact, I left it.  In the line above, I simply added the check for sys.platform=="darwin"